packages/sdk/src/types.ts

```diff
@@ -34,3 +34,12 @@ export interface ExternalAsset extends AssetMetadata {
   internalId: string;
   isWhiteListed?: boolean;
 }
+
+export interface Balance {
+  freeBalance: string;
```

Comment:
can we just use free, total, transferable, .... without balance suffix .. some props have it some not .. don't like it

Reply:
done